subsys/net/lib/aws_iot/src/aws_iot.c

```diff
@@ -1079,6 +1104,14 @@ int aws_iot_init(const struct aws_iot_config *const config,
 		return -ENODATA;
 	}
 
+#if defined(CONFIG_AWS_IOT_BROKER_HOST_NAME_APP)
+	if (IS_ENABLED(CONFIG_AWS_IOT_BROKER_HOST_NAME_APP) &&
+		config->host_name == NULL) {
```

Comment:
Alignment (please align to the opening bracket, otherwise the `if ()` internals blend with the code block).